flake.nix

```diff
@@ -40,7 +40,7 @@
 
                     # ethereum
                     foundry-bin
-                    go-ethereum
+                    # go-ethereum
```

Comment:
@doubledup can you try bump the version in Nix again please.

Reply:
Actually @yrong, is it necessary to upgrade to go-ethereum 1.12? Can we wait for a bit longer?

Reply:
Seems nix packages for 1.12 is not ready yet, For upstreams components(e.g. Polkadot,Cumulus,Geth,Lodestar) we heavily depend on I would prefer to sync more closely to recent release without waiting nix counterpart to be ready. 

Btw, Geth client used here is only for dev setup and I've tested E2E still work as expected. For production usage I think we can still connect to infura endpoint as we did before.

Reply:
ok, sounds fine :+1:

Reply:
Exactly, but I would prefer these locally running clients are only readable for fast api response(reading from public endpoint could be slow), for transactions we can still submit to the infura endpoint as before.